docs/DeveloperGuide.md

```diff
@@ -665,107 +686,118 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+1. User requests to find a patient by name.
+2. DocTrack shows the patient details.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. User enters invalid parameters.
+
+    * 1a1. DocTrack shows an error message.
+
+      Use case resumes at step 1.
+
+* 2a. The patient is not found.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+<br>
 
-    * 3a1. DocTrack shows an error message.
+#### Use case (UC06): Clear all patients 
 
-      Use case resumes at step 2.
+**MSS**
 
-* 3b. The appointment details are invalid (i.e. wrongly formatted or overlap with existing appointment)
+1. User requests to clear all patients.
+2. DocTrack clears all patients.
 
-    * 3b1. DocTrack shows an error message.
+    Use case ends.
 
-      Use case resumes at step 2.
+<br>
 
-**Use case: Duplicate an appointment**
+#### Use case (UC07): Add an appointment
 
 **MSS**
 
-1.  User requests to list appointments.
-2.  DocTrack shows a list of appointments.
-3.  User requests to duplicate a specific appointment in the list on a new date.
-4.  DocTrack duplicates appointment.
+1. User requests to add an appointment.
+2. DocTrack adds the appointment.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. User enters invalid parameters.
 
-  Use case ends.
+    * 1a1. DocTrack shows an error message.
 
-* 3a. The given index is invalid.
+      Use case resumes at step 1.
 
-    * 3a1. DocTrack shows an error message.
+* 1b. User enters an appointment that already exists.
 
-      Use case resumes at step 2.
+    * 1b1. DocTrack shows an error message.
 
-* 3b. The appointment details are invalid (i.e. wrongly formatted or overlap with existing appointment)
+      Use case resumes at step 1.
 
-    * 3b1. DocTrack shows an error message.
-
-      Use case resumes at step 2.
+    
+<br>
 
-**Use case: Update an appointment**
+#### Use case (UC08): Edit an appointment
 
 **MSS**
 
-1.  User requests to list appointments.
-2.  DocTrack shows a list of appointments.
-3.  User requests to update a specific appointment in the list with new details
-4.  DocTrack updates the appointment.
+1. DocTrack shows a list of appointments [(UC10)](#use-case-uc10-view-all-appointments).
+2. User requests to edit a specific appointment in the list with new details.
+3. DocTrack updates the appointment with the new details.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 2a. The given index is invalid.
 
-  Use case ends.
+    * 2a1. DocTrack shows an error message.
 
-* 3a. The given index is invalid.
+      Use case resumes at step 2.
+* 2b. The new appointment details are invalid.
 
-    * 3a1. DocTrack shows an error message.
+    * 2b1. DocTrack shows an error message.
 
       Use case resumes at step 2.
 
-* 3b. The new details are invalid (i.e. wrongly formatted).
+<br>
 
-    * 3b1. DocTrack shows an error message.
+#### Use case (UC09): Delete an appointment
 
-      Use case resumes at step 2.
+**MSS**
+
```

Comment:
Thanks for adding the list to the edit command, but do you think we can add it to delete appointment and person as well? (Since it's a similar use case where the user might want to see the index first)